test/functional/driver/driver-e2e-specs.js

```diff
@@ -83,30 +83,6 @@ describe('XCUITestDriver', function () {
         simsAfter.should.equal(simsBefore);
       });
 
-      it('with udid: uses sim and shuts it down afterwards', async () => {
```

Comment:
@jlipps why it should be? the simulator is already precreated before. It will only create a new one in _driver.init_ if it does not exist yet.